pkg/utils/perm_windows.go

```diff
@@ -6,6 +6,21 @@
 
 package utils
 
+// FileOwner is the ownership a file should have.
+type FileOwner struct {
+	UID string
+	GID string
+}
+
+// CurrentFileOwner returns the executing UID and GID of the current process.
+func CurrentFileOwner() FileOwner {
+	// TODO(blakerouse): Make this return the current user and group on Windows.
```

Comment:
Is this TODO to be implemented in this PR ?

Reply:
It is for a follow up that I am working on, which enabled non Administrator on Windows.